nav2_costmap_2d/CHANGELOG.rst

```diff
@@ -1,12 +1,12 @@
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Changelog for package costmap_2d
+Changelog for package nav2_costmap_2d
```

Comment:
Looks like a global search and replace went too far here, I think these changes in the CHANGELOG.rst should be backed out.

Reply:
Will do

Reply:
As this is a new package, I cleared the change log and noted the direct port from costmap_2d with the exact version number and date

Reply:
Makes sense to me. Thanks!